.github/workflows/nightly-pipeline.yaml

```diff
@@ -10,21 +10,26 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  build-new-node:
-    name: Build node and runtime artifacts (PR version)
+  build-new-production-node:
+    name: Build production node and runtime artifacts (PR version)
     uses: ./.github/workflows/_build-production-node-and-runtime.yml
     secrets: inherit
 
+  build-new-test-node:
```

Comment:
Not sure why we need that step. Nightly should run on production binary, not test one.